iceoryx_examples/waitset_in_c/ice_c_waitset_timer_driven_execution.c

```diff
@@ -129,15 +135,17 @@ int main()
                 iox_notification_info_call(notification);
             }
         }
+        //! [handle events]
     }
+    //! [event loop]
 
-    // cleanup all resources
+    //! [cleanup all resources]
```

Comment:
Maybe move this before the deinit.

Reply:
Then we won't see the Windows exception in the readme

Reply:
Should be fixed now. The thread creation and joining is moved to a separate function.